cost/aws/instance_cloudwatch_utilization/aws_instance_cloudwatch_utilization.pt

```diff
@@ -1,10 +1,17 @@
-name "AWS Instance CloudWatch Utilization"
+name "AWS Inefficient Instance Utilization using CloudWatch"
 rs_pt_ver 20180301
 type "policy"
-short_description "Gathers AWS CloudWatch data for instances on 30 day intervals. See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/aws/instance_cloudwatch_utilization/) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
-long_description "Version: 1.5"
+short_description "Checks inefficient instance utilization using provided CPU and Memory thresholds. Instances matching the criteria can be resized after user approval. See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/aws/instance_cloudwatch_utilization/) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
+long_description ""
 severity "low"
 category "Cost"
+tenancy "single"
```

Comment:
this can be removed.  the credential declaration implies single tenancy.

Reply:
Removed tenancy field from the template